src/client/app/containers/UIOptionsContainer.js

```diff
@@ -14,9 +14,24 @@ import { fetchMetersDataIfNeeded } from '../actions/meters';
  */
 function mapStateToProps(state) {
 	const sortedMeters = _.sortBy(_.values(state.meters.byMeterID).map(meter => ({ id: meter.id, name: meter.name.trim() })), 'name');
+
+	const timeInterval = state.graph.timeInterval;
+	const data = { datasets: [] };
+	for (const meterID of state.graph.selectedMeters) {
+		const readingsData = state.readings.byMeterID[meterID][timeInterval];
```

Comment:
As of right now, this is completely correct. However, when the bar chart is added, readings will be retrieved from state.[line OR bar].readings...
Will the intention be to only grab line chart readings, bar chart readings, or both? If both, some additional logic will eventually be required here.